__tests__/restore.test.ts

```diff
@@ -116,7 +116,7 @@ test("restore on GHES with AC available ", async () => {
 
     expect(stateMock).toHaveBeenCalledWith("CACHE_KEY", key);
```

Comment:
We should have test for restoreOnly where it should be checked that setOutput has been called with CACHE_KEY